helm-charts/strimzi-kafka-operator/README.md

```diff
@@ -44,12 +44,13 @@ The command removes all the Kubernetes components associated with the operator a
 
 ## Configuration
 
-The following table lists the configurable parameters of the Strimzi chart and their default values.  Runtime 
+The following table lists the configurable parameters of the Strimzi chart and their default values.  Runtime
 configuration of Kafka and other components are defined within their respective Custom Resource Definitions.  See
 the documentation for more details.
 
 | Parameter                            | Description                               | Default                                              |
 | ------------------------------------ | ----------------------------------------- | ---------------------------------------------------- |
+| `strimzi.watch_namespaces`           | Comma separated list of namespaces to watch | namespace where helm is applied                    |
```

Comment:
I suggest updating the default here to:

> The namespace that the Helm Release is installed into.

Reply:
thats is not correct :(  `strimzi.watch_namespaces` is a list of namespaces strimzi operator watches, not the namespace where strimzi is deployed into.

Given for example the following helm command

```helm upgrade test --install  ./helm-charts/strimzi-kafka-operator/ --debug  --dry-run --namespace=strimzi-operator-ns --set strimzi.watch_namespaces="{yyy,xxx}"```

strimzi-cluster-operator would be deployed into strimzi-operator-ns and its configuration value would be:

```
          env:
            - name: STRIMZI_NAMESPACE
              value: yyy,xxx,strimzi-operator-ns
```

Let me know if still unclear please :)

Reply:
I see.  That makes sense.  Using your example, I think it should be made clear that the default setting (if namespaces `yyy,xxx` weren't defined) that Strimzi would be watching the namespace the operator is deployed into.  Otherwise the user may think that not defining this value will mean the operator watches no namespaces.